package.json

```diff
@@ -34,26 +34,26 @@
     "dependencies": {
         "@autorest/codemodel": "^4.15.0",
         "@autorest/extension-base": "^3.2.1",
-        "@azure-rest/core-client": "^1.0.0-beta.7",
         "@azure-tools/codegen": "^2.5.294",
         "@azure/core-client": "^1.3.0",
         "@azure/core-http": "^1.2.4",
         "@azure/core-lro": "^2.1.0",
         "@azure/core-paging": "^1.2.0",
-        "@azure/core-tracing": "^1.0.0-preview.12",
         "@azure/core-rest-pipeline": "^1.3.0",
+        "@azure/core-tracing": "^1.0.0-preview.12",
         "@azure/logger": "^1.0.0",
         "@opentelemetry/api": "^0.10.2",
         "@types/lodash": "^4.14.149",
+        "fs-extra": "^10.0.0",
         "handlebars": "^4.7.7",
         "lodash": "^4.17.21",
         "prettier": "^1.19.1",
         "source-map-support": "^0.5.16",
-        "ts-morph": "^9.0.0",
-        "fs-extra": "^10.0.0"
+        "ts-morph": "^9.0.0"
     },
     "devDependencies": {
         "@autorest/test-server": "^3.0.27",
+        "@azure-rest/core-client": "1.0.0-beta.8",
```

Comment:
Why are we moving this to devDependencies?

Reply:
It's a mistake, let me change it back